integrasjon/rest-klient/src/main/java/no/nav/vedtak/felles/integrasjon/rest/tokenhenter/OidcTokenResponse.java

```diff
@@ -0,0 +1,6 @@
+package no.nav.vedtak.felles.integrasjon.rest.tokenhenter;
+
+public record OidcTokenResponse(String access_token,
```

Comment:
Override toString for å maskere hele eller deler av access_token ved eventuell logging?

Reply:
Done